definitions/checks/check_capsule_package.rb

```diff
@@ -0,0 +1,19 @@
+module Checks
+  module Foreman
+    class Checks::CheckCapsulePackage < ForemanMaintain::Check
```

Comment:
@patilsuraj767, do we still need this check?

Reply:
@kgaikwad It is useful in case of the satellite to check if the capsule package is installed but in case of `foreman-maintain upgrade check` command it comes useless because foreman-maintain will fail with `Too many scenarios match {:tags=>[:upgrade_scenario, :pre_upgrade_checks]} `  way before this check get execute. 
Should I remove the check?

Reply:
Are you still facing `Too many scenarios` error after confine block change related to satellite package checking at capsule feature?

Reply:
Yeah, that confine block helped in not detecting capsule feature on satellite if satellite-capsule package is installed. Did not faced `Too many scenarios` error. I will remove the check.